packages/@okta/i18n/src/properties/login_cs.properties

```diff
@@ -1302,7 +1306,6 @@ oie.registration.form.update.submit = Odeslat
 oie.registration.form.customize.label = {0}
 ## {0} is the form submit button label customized by admin
 oie.registration.form.customize.buttonLabel = {0}
-oie.user.profile.login = U\u017Eivatelsk\u00E9 jm\u00E9no
```

Comment:
@pengyao-okta This property was incorrectly removed in a previous change and was put back recently in https://github.com/okta/okta-signin-widget/commit/c5e08948a75c8e51cb413f58a3bc5328c929e9df

A couple of questions:

It's currently in `login.properties` in `master` but perhaps the translation pull occurred before it was put back. Is there anything we need to do to ensure it isn't deleted in future translations?

Can we manually remove these in this branch and merge or should we let the translation script run again to generate a new PR?